FAQ.md

```diff
@@ -30,40 +30,53 @@ We like getting inspired on [codepen.io](https://codepen.io) and building protot
 
 ## Technical FAQ
 
-### I don't know how to test my module within the COBI.bike app
+### I don't know how to test my module within the COBI.Bike app
 
 Did you already sign up as a developer on [my.cobi.bike](https://my.cobi.bike)? If yes, please follow the instructions on the [DevKit site](https://github.com/cobi-bike/COBI-DevKit).
 
 ### Sometimes the settings of my module are reset
 
-If your iOS device is low on storage, the Safari browser cache may get deleted. This is a known issue for now. 
+If your iOS device is low on storage, the Safari browser cache may get deleted. This is a known issue for now.
 
 ### I'm using the `subscribe` method, but I don't receive any data
 
 Make sure to authenticate your module first – to do so call
 `COBI.init("YOUR-TOKEN")` before you subscribe to any `COBI.js` data stream.
 
-### Any tips for debugging my module within the COBI.bike app on the phone?
+### I changed the code of the module but the changes are not appearing on the COBI.Bike app
+
+We have noticed that sometimes there are some browser-related caching issues so this is what might be happening to you. The easiest solution would be to go into the module selection screen, remove *My Module* and add it again.
+
+### Any tips for debugging my module within the COBI.Bike app on the phone?
 
 The native app will display javascript errors in a dialog. Just activate this feature in the diagnostics settings of the app.
```

Comment:
it would be nice to add a link to a documentation site here

Reply:
it would but all there is as far as I can tell are reddit/stackoverflow/blog posts. Nothing official from Apple, if that's what you mean.

Reply:
no I mean the diagnostic settings in the app. I think this refers to the COBI app right ?